home.md

```diff
@@ -2,33 +2,33 @@
 
 The way data is currently shared and published via academic papers, at least from genetic analyses, is mainly via releasing raw sequencing data into public repositories such as the [ENA](https://www.ebi.ac.uk/ena), while providing partial metadata on samples via poorly formatted Excel tables in the Supplement. This creates (at least) the following problems:
 
-1. Intermediate data such as genotypes or metagenomic profiles are often not released at all, making it hard for others to reproduce analyses. 
+1. Intermediate data such as genotypes are often not released at all, making it hard for others to reproduce analyses. 
 2. The connection between individuals, contextual information, and genetic data becomes hard to maintain, bridging between very different repositories and sources (Excel vs. personal homepages vs. public repositories)
-4. Meta-analyses spanning datasets require enormous amounts of work on data collection and curation.
-5. Studies that combine data from different technologies (isotopes, C14 dating, genetics) have no clear way to release such complex relationships in a concise way.
-6. Incrementally produced data, for example by adding new data to  previously published individuals, cannot be easily connected to the same individuals.
+3. Meta-analyses spanning datasets require enormous amounts of work on data collection and curation.
+
+A major initiative to address these problems is the [Allen Ancient DNA Resource](https://reich.hms.harvard.edu/allen-ancient-dna-resource-aadr-downloadable-genotypes-present-day-and-ancient-dna-data) ("AADR"), which is a curated dataset of public ancient DNA data generated, curated and bundled by David Reich's ancient DNA laboratory at Harvard University. In many ways, our initiative is inpiried by and deriving from this resource. In particular, the AADR currently (March 2021) is arguably the most complete resource world-wide that provides genome-wide genotype data for ancient individuals from nearly all publications in the field.
+
+As detailed under [Repositories](repos), our public repository derives to a large extent directly from the AARD, while many curated packages, in particular from 2019 and later, contain data compiled and generated by us. Our initiative differs in one further important aspect from the AARD: Instead of providing the data in one monolithic dataset, we decided to structure the data in terms of packages, which typically (but don't have to) contain data from single publications. This makes it much simpler for us to keep the resource up to date, as new packages can simply be added, and patches to individual packages are light-weight and don't necessitate an update of the entire dataset. To facilitate working with this more de-centralised package-oriented system, we provide our command line utility [trident](trident), which is provided as pre-compiled executable for all major operating systems and is continuously developed and maintainted.
```

Comment:
No `/repos` any more.

I would argue the guaranteed machine-readability of context data in .janno files is a second major difference to the AADR. Automatic validation is clearly lacking there - despite the excellent content curation.

Reply:
True, good to put that in. I'll work on that tomorrow.